### PR TITLE
TRADE-138 - Configure batchee stitchee for testing

### PR DESCRIPTION
**Pull request by danielfromearth**
## Jira Issue ID

TRADE-138

## Description

This PR adds configurations for the Batchee>Stitchee>Concise, and the L2ss>Batchee>Stitchee>Concise service chains.

## Local Test Steps

- Pull this branch of Harmony locally.
- Pull the Docker images for Batchee and Stitchee from GitHub, or build them locally from the HEAD of main for the Batchee and Stitchee repositories.
- Run through the regression tests locally, to make sure the service gets invoked.

## PR Acceptance Checklist
* [ ] Acceptance criteria met
* [ ] Tests added/updated (if needed) and passing
* [ ] Documentation updated (if needed)

### CODE REVIEW

**Comment by danielfromearth** [comment]
@chris-durbin, this PR of configuration changes for Batchee>Stitchee service chains is from my and @ank1m 's local tests of the new service chain configurations. It's probably not totally ready for merging yet, but I thought creating this PR would give us a new space to further work out the details.

File: config/services.yml [hunk 1]
```diff
@@ -1021,3 +1021,82 @@ https://cmr.uat.earthdata.nasa.gov:
...
+        operations: ['extend']
+      - image: !Env ${PODAAC_CONCISE_IMAGE}
+        is_batched: true
+        operations: ['concatenate']
+        conditional:
+          exists: ['concatenate']
+
+  - name: l2-subsetter-batchee-stitchee-concise
+    description: |
+      TODO - fill in description
```
Comment (chris-durbin) [inline config/services.yml:1064]: Add a description for this service chain.

File: config/services.yml [hunk 2]
```diff
@@ -1021,3 +1021,82 @@ https://cmr.uat.earthdata.nasa.gov:
...
+    data_operation_version: '0.18.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/asdc-trade/l2-subsetter-batchee-stitchee-concise
+    umm_s: ''
```
Comment (chris-durbin) [inline config/services.yml:1073]: Needs a UMM-S record.

**Review by chris-durbin — COMMENTED** [review]
I'll test out a deployment to a harmony sandbox environment.

File: config/services.yml (hunk 2 above)
Comment (danielfromearth) [reply config/services.yml:1073]: working now on setting up a UMM-S for this second chain.

**Comment by danielfromearth** [comment]
I am not sure what to make of the errors in these checks, @chris-durbin. 
E.g., `Concatenation mismatch: harmony is true and UMM-S is false.`.  Does that point to something in the files changed in this PR, or is it something else?

**Review by chris-durbin — APPROVED** [review]
Tested successfully in a sandbox environment along with the harmony-1599 branch in the harmony-ci-cd repo. Will merge both PRs in together at the same time.

**Comment by chris-durbin** [comment]
> I am not sure what to make of the errors in these checks, @chris-durbin. E.g., `Concatenation mismatch: harmony is true and UMM-S is false.`. Does that point to something in the files changed in this PR, or is it something else?

Those failing checks were for different UMM-S records (not the one you added). There are separate tickets to resolve those discrepancies - your service configuration reported no issues.

**Review by indiejames — APPROVED** [review]

**Review by ygliuvt — APPROVED** [review]

**Pull request merged**